tests/integ/modin/frame/test_set_index.py

```diff
@@ -312,15 +319,23 @@ def test_set_index_pass_arrays_duplicate(obj_type1, obj_type2, drop, append, nat
     elif obj_type2 == pd.Index:
         obj_type2 = native_pd.Index
     native_keys = [obj_type1(array), obj_type2(array)]
-    eval_snowpark_pandas_result(
-        snow_df,
-        native_df,
-        lambda df: df.set_index(
-            native_keys if isinstance(df, native_pd.DataFrame) else keys,
-            drop=drop,
-            append=append,
-        ),
-    )
+
+    query_count = 4
+    # one extra query per modin index to create the series and set index
+    if obj_type1 == native_pd.Index and obj_type2 == native_pd.Index:
+        query_count = 6
+    elif obj_type1 == native_pd.Index or obj_type2 == native_pd.Index:
+        query_count = 5
```

Comment:
This is a nit but one way you could make this more concise is:
```py
query_count = 4
query_count += 1 if obj_type1 == native_pd.Index else 0
query_count += 1 if obj_type2 == native_pd.Index else 0
```
since each obj adds 1 query if it's pandas